com.github.paulwedeck.settlers-remake.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.github.paulwedeck.settlers-remake
```

Comment:
we are enforcing the rule that say the app-id for github hosted starts with `io.github` and not `com.github`